.github/workflows/django_tests_against_emulator0.yml

```diff
@@ -0,0 +1,32 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+name: django-tests0
+jobs:
+  system-tests:
+    runs-on: ubuntu-latest
+
+    services:
+      emulator-0:
+        image: gcr.io/cloud-spanner-emulator/emulator:latest
+        ports:
+          - 9010:9010
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
```

Comment:
We're using python 3.8 in emulator tests but 3.6 in `django_test_suite.sh`?

Reply:
I think it's done this way because version of kokoro Linux includes Python 3.6. But in workflows we can set Python fast and clean.